hadoop.md

```diff
@@ -0,0 +1,5 @@
+# Hadoop
+The platform uses Hadoop HDFS as a file system for Spark. For web-ui access to it use the following URL.
+```
+http:<HOST_NAME>0.<HOST_DOMAIN>:50070
```

Comment:
Unclear which hostname

Maybe add that the web-ui is available on the hadoop-namenode specified in the inventory file.